Dockerfile

```diff
@@ -3,7 +3,7 @@
 # 
 # TO OPT OUT OF UPDATES, SEE https://github.com/lyft/boilerplate/blob/master/Readme.rst
 
-FROM golang:1.17.1-alpine3.14 as builder
+FROM golang:1.18-alpine3.16 as builder
```

Comment:
Does alpine offer an image with go 1.19?

Reply:
yes, synk has opened up PRs to bump our alpine 3.16 images to 3.17 so the plan is to address the golang version bump in our dockerfiles alongside those updates.